src/main.cc

```diff
@@ -348,8 +348,11 @@ int main( int argc, char ** argv )
 
 
   //high dpi screen support
-  qputenv( "QT_ENABLE_HIGHDPI_SCALING", "1" );
-  QApplication::setHighDpiScaleFactorRoundingPolicy( Qt::HighDpiScaleFactorRoundingPolicy::PassThrough );
+  if (!qEnvironmentVariableIsSet("QT_ENABLE_HIGHDPI_SCALING") ||
+       qEnvironmentVariableIsEmpty("QT_ENABLE_HIGHDPI_SCALING")) {
+    qputenv( "QT_ENABLE_HIGHDPI_SCALING", "1" );
+    QApplication::setHighDpiScaleFactorRoundingPolicy( Qt::HighDpiScaleFactorRoundingPolicy::PassThrough );
```

Comment:
should this line put outside of the if block.

Reply:
Done.